libs/core/src/lib/avatar/avatar.component.scss

```diff
@@ -0,0 +1,47 @@
+@import '~fundamental-styles/dist/avatar';
+
+// Temporary style fix until the latest version of Fundamental-styles is released
+fd-avatar[role='button'] {
+    cursor: pointer;
+}
+
+.fd-avatar--accent-color-1 {
```

Comment:
Don't we have them on styles?

Reply:
We do, but it's not released yet. The current styles version (in ngx) uses `has-color-`